gslib/utils/boto_util.py

```diff
@@ -392,6 +400,66 @@ def _PatchedGetPluginMethod(cls, requested_capability=None):
 
   boto.plugin.get_plugin = _PatchedGetPluginMethod
 
+  #########################################################################
```

Comment:
We don't need this method patch anymore since we're using our updated version of Boto.